examples/simple-udp/server/main.go

```diff
@@ -29,7 +29,7 @@ import (
 
 // main starts a UDP server that received 1024 byte sized packets at at time
 // converts the bytes to a string, and logs the output
-func main() {
+func main() { // nolint: gocyclo
```

Comment:
Same here, we should make a ticket to fix this one too.

Reply:
Created https://github.com/GoogleCloudPlatform/agones/issues/178